utils/index.js

```diff
@@ -192,6 +192,16 @@ const currencyFilter = (number, fractionDigits = 2) => {
   })
 }
 
+const sortByLineNumber = (...objToSort) => {
+  const superObj = Object.assign({}, ...objToSort);
+  const filteredObj = Object.entries(superObj).filter(obj => {
+    return typeof obj[1] === 'object' && obj[1] !== null && obj[1] !== undefined && obj[1].hasOwnProperty('line')
+  })
+  const sortedArrayObj = filteredObj.map(obj => obj[1]).sort((a,b) => a.line - b.line)
```

Comment:
Nice! So if it hits value that's not an object or doesn't have the "line" key, it will get filtered out, right?

Reply:
yup!